pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1289,6 +1289,11 @@ public void deleteLedgerComplete(Object ctx) {
                 });
 
                 return deleteFuture;
+                }).whenComplete((value, ex) -> {
```

Comment:
nit: you could use 'exceptionally'

Reply:
exceptionally swallows the exception so it's not a good fit here